<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#loginDialog{
		background-image: url(:/images/Login.png);
}</string>
  </property>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>300</width>
     <height>58</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#usernameLabel{	
background-image: url(:/images/textborder.png);
color:#a2a2a2;
	font:  22px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>168</y>
     <width>288</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#usernameEdit{
	border-style:none;
	border-radius: 13px;
	background: transparent;
	color:#2196f3;
	font:  22px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>300</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#passwordLabel{	
background-image: url(:/images/passwordborder.png);
color:#a2a2a2;
	font:  22px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>260</y>
     <width>286</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#passwordEdit{
	border-style:none;
	border-radius: 13px;
	background: transparent;
	color:#2196f3;
	font:  22px;
	lineedit-password-character: 9679;
}
QLineEdit:echoMode {
    lineedit-password-character: 9679;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label{
		color:#feffff;
		font:bold 40px;
}</string>
   </property>
   <property name="text">
    <string>The  App</string>
   </property>
  </widget>
  <widget class="QWidget" name="titlebar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="minbutton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#minbutton{
	border-image:url(:/images/minibutton_normal.png);border:0px;
} 
QPushButton#minbutton:hover{
	border-image:url(:/images/minbutton_over.png);border:0px
} 
QPushButton#minbutton:pressed{
	border-image:url(:/images//minibutton_on.png); position: relative;top: 1px; left: 1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closebutton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#closebutton{
	border-image:url(:/images/closebutton_normal.png);border:0px;
} 
QPushButton#closebutton:hover{
	border-image:url(:/images/close_over.png);border:0px
} 
QPushButton#closebutton:pressed{
	border-image:url(:/images//close_on.png); position: relative;top: 1px; left: 1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_2{
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>title</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>148</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton{
	border-style:none;
	border-image: url(:/images/sign_normal.png);
	border-radius: 13px;
	background: transparent;
	color:#2196f3;
	font:  22px;
	lineedit-password-character: 9679;
}
QPushButton#pushButton:hover{
	border-style:none;
	border-image: url(:/images/sign_over.png);
	border-radius: 13px;
	background: transparent;
	color:#2196f3;
	font:  22px;
	lineedit-password-character: 9679;
}
QPushButton#pushButton:pressed{
	border-style:none;
	border-image: url(:/images/sign_on.png);
	border-radius: 13px;
	background: transparent;
	color:#2196f3;
	font:  22px;
	lineedit-password-character: 9679;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
